packages/mobile/src/send/SendConfirmation.tsx

```diff
@@ -245,7 +245,7 @@ function SendConfirmation(props: Props) {
     )
   }
 
-  const allowComment = !isInvite && tokenInfo?.isCoreToken
+  const allowComment = !isInvite && tokenInfo?.isCoreToken && tokenInfo.symbol !== 'CELO'
```

Comment:
Is there another place where we do a similar check?
Wondering if we should turn this into a small helper.

Reply:
I think there are a couple yeah, I can do that